crates/modelardb_server/src/manager.rs

```diff
@@ -106,7 +106,26 @@ impl Manager {
 
         // For each table to create, register and save the table in the metadata Delta Lake.
         for create_table_sql in create_table_sql_commands {
-            parser::tokenize_and_parse_sql_statement(create_table_sql)?;
+            match parser::tokenize_and_parse_sql_statement(create_table_sql)? {
+                ModelarDbStatement::CreateNormalTable { name, schema } => {
+                    context
+                        .create_normal_table(name, schema, create_table_sql)
+                        .await?;
+                }
+                ModelarDbStatement::CreateModelTable(model_table_metadata) => {
+                    context
+                        .create_model_table(model_table_metadata, create_table_sql)
+                        .await?;
+                }
+                ModelarDbStatement::Statement(_)
+                | ModelarDbStatement::IncludeSelect(..)
+                | ModelarDbStatement::DropTable(_)
+                | ModelarDbStatement::TruncateTable(_) => {
+                    return Err(ModelarDbServerError::InvalidArgument(
```

Comment:
Maybe change this to be an internal error since this error is not caused by the user providing an invalid argument.

Reply:
Fixed by changing the error to `ModelarDbServerError::InvalidState`.